src/Dodo.HttpClient.ResiliencePolicies/RetryPolicy/RetryAfterExtension.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Dodo.HttpClientResiliencePolicies.RetryPolicy
+{
+	public static class RetryAfterExtension
```

Comment:
If we remove EnableRetryAfterFeature, so we can get rid of this thing too.

Reply:
removed.